lonestar/mining/cpu/triangle-counting/README.md

```diff
@@ -9,21 +9,6 @@ This program counts the number of triangles in a given undirected graph.
 INPUT
 --------------------------------------------------------------------------------
 
-We support the following four input graph formats: **gr**, **txt**, **adj**, **mtx**.
-
-We mostly use **adj** format as also used by Arabesque and RStream.
-The **adj** format takes as input graphs with the following formats:
-
-* **Graphs label on vertex(default)**
-```
-# <num vertices> <num edges>
-<vertex id> <vertex label> [<neighbour id1> <neighbour id2> ... <neighbour id n>]
-<vertex id> <vertex label> [<neighbour id1> <neighbour id2> ... <neighbour id n>]
-...
-```
-
-Vertex ids are expected to be sequential integers from 0 to (total number of vertices - 1).
-
 This application takes in symmetric and simple Galois .gr graphs.
```

Comment:
Ditto

Reply:
I didn't add this. @nicelhc13 Is this consistent with others?